### PR TITLE
Add more docs on adding ITs

### PR DESCRIPTION
**Pull request by damccorm**
Expands the load test doc to also talk about ITs

Part of #1459

### CODE REVIEW

**Comment by liferoad** [comment]
@Abacn please take a look. Thanks.

File: contributor-docs/add-integration-or-load-test.md [hunk 1]
```diff
@@ -423,6 +462,9 @@ public void testBacklog() {
   TableId table = bigQueryResourceManager.createTable(testName, SCHEMA);
   
   // Generate fake data to Pub/Sub topic
+  // In a normal integration test (non-load test), you can use the resource manager
```
Comment (Abacn) [inline contributor-docs/add-integration-or-load-test.md:465]: how about

```suggestion
  // In a normal integration test (process a small amount of data), you can use the resource manager
```

same below

**Review by Abacn — COMMENTED** [review]
Thanks!

Just had a minor comment

File: contributor-docs/add-integration-or-load-test.md (hunk 1 above)
Comment (damccorm) [reply contributor-docs/add-integration-or-load-test.md]: SGTM, updated

**Review by Abacn — APPROVED** [review]

**Pull request merged**